c/ssh-simplified/s3_srvr_1_true-unreach-call.cil.c

```diff
@@ -5,6 +5,9 @@ extern int __VERIFIER_nondet_int(void);
 extern long __VERIFIER_nondet_long(void);
 extern void *__VERIFIER_nondet_pointer(void);
 extern int __VERIFIER_nondet_int();
+
+#include <stdio.h>
```

Comment:
It's actually only due to a `printf("error")` statement in the error function. Are these outputs used for anything? Otherwise, should I just remove the printf statement?